backend/src/db/seed.js

```diff
@@ -823,119 +823,140 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
       louie.relateTo(p10, 'shouted'),
     ])
 
-    const reports = await Promise.all([
-      Factory.build('report'),
-      Factory.build('report'),
-      Factory.build('report'),
-      Factory.build('report'),
-    ])
-    const reportAgainstDagobert = reports[0]
-    const reportAgainstTrollingPost = reports[1]
-    const reportAgainstTrollingComment = reports[2]
-    const reportAgainstDewey = reports[3]
-
     // report resource first time
-    await Promise.all([
-      reportAgainstDagobert.relateTo(jennyRostock, 'filed', {
-        resourceId: 'u7',
-        reasonCategory: 'discrimination_etc',
-        reasonDescription: 'This user is harassing me with bigoted remarks!',
-      }),
-      reportAgainstDagobert.relateTo(dagobert, 'belongsTo'),
-      reportAgainstTrollingPost.relateTo(jennyRostock, 'filed', {
-        resourceId: 'p2',
-        reasonCategory: 'doxing',
-        reasonDescription: "This shouldn't be shown to anybody else! It's my private thing!",
-      }),
-      reportAgainstTrollingPost.relateTo(p2, 'belongsTo'),
-      reportAgainstTrollingComment.relateTo(huey, 'filed', {
-        resourceId: 'c1',
-        reasonCategory: 'other',
-        reasonDescription: 'This comment is bigoted',
-      }),
-      reportAgainstTrollingComment.relateTo(trollingComment, 'belongsTo'),
-      reportAgainstDewey.relateTo(dagobert, 'filed', {
-        resourceId: 'u5',
-        reasonCategory: 'discrimination_etc',
-        reasonDescription: 'This user is harassing me!',
-      }),
-      reportAgainstDewey.relateTo(dewey, 'belongsTo'),
+    const [
+      reportAgainstDagobert,
+      reportAgainstTrollingPost,
+      reportAgainstTrollingComment,
+      // reportAgainstDewey,
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-12T16:38:28Z" title="Wednesday, February 12th 2020, 5:38:28 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -823,119 +823,140 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
       louie.relateTo(p10, 'shouted'),
     ])
 
-    const reports = await Promise.all([
-      Factory.build('report'),
-      Factory.build('report'),
-      Factory.build('report'),
-      Factory.build('report'),
-    ])
-    const reportAgainstDagobert = reports[0]
-    const reportAgainstTrollingPost = reports[1]
-    const reportAgainstTrollingComment = reports[2]
-    const reportAgainstDewey = reports[3]
-
     // report resource first time
-    await Promise.all([
-      reportAgainstDagobert.relateTo(jennyRostock, 'filed', {
-        resourceId: 'u7',
-        reasonCategory: 'discrimination_etc',
-        reasonDescription: 'This user is harassing me with bigoted remarks!',
-      }),
-      reportAgainstDagobert.relateTo(dagobert, 'belongsTo'),
-      reportAgainstTrollingPost.relateTo(jennyRostock, 'filed', {
-        resourceId: 'p2',
-        reasonCategory: 'doxing',
-        reasonDescription: "This shouldn't be shown to anybody else! It's my private thing!",
-      }),
-      reportAgainstTrollingPost.relateTo(p2, 'belongsTo'),
-      reportAgainstTrollingComment.relateTo(huey, 'filed', {
-        resourceId: 'c1',
-        reasonCategory: 'other',
-        reasonDescription: 'This comment is bigoted',
-      }),
-      reportAgainstTrollingComment.relateTo(trollingComment, 'belongsTo'),
-      reportAgainstDewey.relateTo(dagobert, 'filed', {
-        resourceId: 'u5',
-        reasonCategory: 'discrimination_etc',
-        reasonDescription: 'This user is harassing me!',
-      }),
-      reportAgainstDewey.relateTo(dewey, 'belongsTo'),
+    const [
+      reportAgainstDagobert,
+      reportAgainstTrollingPost,
+      reportAgainstTrollingComment,
+      // reportAgainstDewey,
```


Why you removed the `reportAgainstDewey`? @roschaefer